src/App.vue

```diff
@@ -1,54 +1,54 @@
 <template>
-  <v-app
-    :dark="isDarkTheme"
-    class="application--linear-gradient"
-  >
+  <v-app :theme="themeName" class="application--linear-gradient">
     <warning-on-addresses-dialog v-model="showWarningOnAddressesDialog" />
     <component :is="layout">
       <router-view />
     </component>
   </v-app>
 </template>
 
-<script>
+<script lang="ts">
+import { defineComponent } from 'vue'
 import dayjs from 'dayjs'
-import WarningOnAddressesDialog from '@/components/WarningOnAddressesDialog'
+import WarningOnAddressesDialog from '@/components/WarningOnAddressesDialog.vue'
 import Notifications from '@/lib/notifications'
+import { ThemeName } from './plugins/vuetify'
 
-export default {
+export default defineComponent({
   components: {
     WarningOnAddressesDialog
   },
   data: () => ({
-    showWarningOnAddressesDialog: false
+    showWarningOnAddressesDialog: false,
+    notifications: null as Notifications | null
   }),
   computed: {
-    layout () {
+    layout() {
       return this.$route.meta.layout || 'default'
     },
-    isLogged () {
+    isLogged() {
       return this.$store.getters.isLogged
     },
-    isDarkTheme () {
-      return this.$store.state.options.darkTheme
-    },
-    isLoginViaPassword () {
+    isLoginViaPassword() {
       return this.$store.getters['options/isLoginViaPassword']
+    },
+    themeName() {
+      return this.$store.state.options.darkTheme ? ThemeName.Dark : ThemeName.Light
     }
   },
-  created () {
+  created() {
     this.setLocale()
   },
-  mounted () {
+  mounted() {
```

Comment:
you removed the spaces here, but not in the other files

Reply:
Recommended ESLint configuration for Vue3 differs from the configuration for Vue2. The file was linted because it was modified. I will leave it as is until we decide which configuration we will stick with. Then we will lint the rest of the codebase in other PR.